LocalWiki.php

```diff
@@ -182,6 +182,14 @@ function wfDonateLink( $skin, &$bar ) {
 	unset( $wgGroupPermissions['checkuser'] );
 }
 
+if ( $wgDBname === 'snapwikiwiki' ) {
+	$wgTranslateDocumentationLanguageCode = 'qqq';
```

Comment:
This can be done via Special: ManageWiki. In addition, this value ("qqq") is the default.

Reply:
qqq is not default.

Reply:
I noticed my mistake.

Reply:
And apologies for not noticing this before.

Reply:
It should still be removed though since can be done in ManageWiki

Reply:
Done.